src/main/java/com/lorepo/icplayer/client/printable/PrintableContentParser.java

```diff
@@ -275,33 +281,49 @@ private String applyHeaderAndFooter(String content, boolean fullHeight) {
 		return result;
 	}
 	
-	private native SplitPageResult splitPageModules(PrintableContentParser x, String html, int pageWidth, int pageHeight) /*-{
+	private native SplitResult splitPageModules(PrintableContentParser x, String html, int pageWidth, int pageHeight) /*-{
 		var pages = "";
 		var $wrapper = $wnd.$("<div></div>");
 		$wrapper.html(html);
 		
+		var SPLITTABLE_CLASS_NAME = @com.lorepo.icplayer.client.printable.PrintableContentParser::SPLITTABLE_CLASS_NAME;
+		var enableTwoColumnPrint  = x.@com.lorepo.icplayer.client.printable.PrintableContentParser::getTwoColumnPrintEnabled()();
+		var maxEmptyHeight = pageHeight * 0.2;

```

Comment:
Skąd bierze się to 0.2? Czy to jakiś warunek biznesowy? Przydałoby się to jakoś okomentować.